src/utils/dispute-utils.js

```diff
@@ -543,7 +547,7 @@ export function getRoundFees(round, courtConfig) {
 /**
  *
  * @param {BigNum} minActiveBalance The minimum active balance required to become an active juror
- * @param {BigNum} penaltyPct Permyriad of min active tokens balance to be locked to each drafted juror
+ * @param {BigNum} penaltyPct Percentage of min active tokens balance to be locked to each drafted juror
```

Comment:
```suggestion
 * @param {BigNum} penaltyPct Percentage (against PCT_BASE) of min active tokens balance to be locked to each drafted juror
```

"Percentage" usually implies something smaller than 100, but we have 10^16 values here :)